rpc/manager/manager.proto

```diff
@@ -237,6 +237,11 @@ message InterceptInfo {
 
 message SessionInfo {
   string session_id = 1;
+  string cluster_id = 2;
+  oneof session {
+    ClientInfo client = 3;
+    AgentInfo agent = 4;
```

Comment:
Why is the `ClientInfo` / `AgentInfo` needed here? Isn't that a lot of unnecessary data?

Reply:
So the idea is that we need the install id to be available, so it'd be included through the ClientInfo.
I guess we could just add an optional installid field instead of ClientInfo but my thinking is this is much more future-proof because agents will not have install ids anyway and this lets us see what kind of session it is

Reply:
Fair enough 👍